crt_portal/cts_forms/templates/forms/report_details.html

```diff
@@ -2,15 +2,20 @@
 {% block form_questions %}
 <p class="margin-top-1 margin-bottom-4"><em>{{ page_note }}</em></p>
 
-<fieldset class="usa-fieldset">
   {% for field in wizard.form.visible_fields %}
     <div class="usa-prose margin-bottom-2">
       <label for="{{ field.id_for_label }}"
              class="big margin-bottom-0">
         {{ field.label }}
       </label>
       <p><em>{{ field.help_text }}</em></p>
-      {{ field.errors }}
+      {% if field.errors %}
+        <div class="usa-alert usa-alert--warning usa-alert--slim">
```

Comment:
Could you say more about the reason for adding an alert div here? There is already an alert div on the `report_details.html` page about ~15 lines below, so this may be introducing a duplicative alert.

Reply:
It's not a new alert, it was already there but not styled the same as the other error fields.

Reply:
I see, thanks! So essentially this alert is for back-end errors and the alert below is for front-end errors. In this case the back-end errors should never be triggered because the field is optional and we're trimming down word count to be below the database constraint ... but if somehow a back-end error does appear, it'll be styled right! Appreciate the explanation.